crates/sui-graphql-rpc/src/server/builder.rs

```diff
@@ -171,8 +171,15 @@ mod tests {
         assert!(resp.is_ok());
 
         // Should timeout
-        let resp = test_timeout(timeout, timeout).await;
-        assert!(resp.is_err());
+        let errs: Vec<_> = test_timeout(timeout, timeout)
+            .await
+            .into_result()
+            .unwrap_err()
+            .into_iter()
+            .map(|e| e.message)
+            .collect();
+        let exp = format!("Request timed out. Limit: {}s", timeout.as_secs_f32());
```

Comment:
Suggestion: I think @amnn added expect-test to this crate, https://docs.rs/expect-test/latest/expect_test/ can use that for any string snapshot comparisons

Reply:
good suggestion but had some issues getting this to work with non-literals

Reply:
w/ expect_test you can just set the expected_str = [""] and let the crate fill in the string for you, for example https://github.com/MystenLabs/sui/blob/main/crates/sui-json-rpc/src/coin_api.rs#L1347 has a bunch of non-literals, but I just deferred to expect_test to populate